### PR TITLE
BLUE 2

### PR DESCRIPTION
**Pull request by anhnt170489**
this PR seemed to be out of date due to being late considered (https://github.com/huggingface/transformers/pull/1238). 

So I updated the code to be able to merge with the latest version. 

In this PR:

- I add BertForMultiLabelClassification, RobertaForTokenClassification, RobertaForMultiLabelClassification.

- I add examples for Finetuning the BERT, RoBERTa models for tasks on BLUE (https://github.com/ncbi-nlp/BLUE_Benchmark). BLUE (Biomedical Language Understanding Evaluation) is similar to GLUE, but for Biomedical data. The "run_blue", "utils_blue" are customized from "run_glue", "utils_glue", but more sufficient, because it contains not only sequence classification, but also token classification, multi-label classification. People may also have more options for examples of fine-tuning BERT/RoBERTa.

- I also add test function to test_examples as well as test data

### CODE REVIEW

**Comment by thomwolf** [comment]
Ok, I think this is great, sorry for the delay in reviewing the PR.

**Comment by thomwolf** [comment]
Do you want to just add the new RoBERTa models in the tests, at least [this line](https://github.com/huggingface/transformers/blob/master/transformers/tests/modeling_roberta_test.py#L38).

Also, optional but could be nice if you feel like it: add TF 2.0 counterparts to your new PyTorch heads (you can just copy-past-adapt) the relevant Bert heads.
